FirebaseSessions/SourcesObjC/NanoPB/FIRSESNanoPBHelpers.m

```diff
@@ -86,7 +94,7 @@
   if (pbBytes == NULL) {
     return NULL;
   }
-  memcpy(pbBytes->bytes, [data bytes], data.length);
+  [data getBytes:pbBytes->bytes length:data.length];
```

Comment:
also here, why the change?

Reply:
This change isn't necessary; I can remove it. I made it while monkeying around in these files and it adheres to the principle of use the highest level of abstraction available so I left it.

Reply:
Got it– I thought it was necessary for the fix. Keeping it SGTM if you prefer.